### DISCUSSION TITLE
How can I run this project?

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [uvdesk/community-skeleton#584] [Q&A] [answered]
Hello,

I hope you're well and everything is good for you.

Does this platform have a team-based assignment?
Can I assign a ticket I created to another team?

How can I run this project? I install every requirement?

Thank you very much.
Best regards,

**Comment by PeopleInside** [comment, marked as answer]
Hi, thank you for your question.
I'm a passionate UVdesk member.

You can try the open source demo at https://demo.uvdesk.com/
You can find more info about the open source version here: https://www.uvdesk.com/it/opensource/ and on the documentation: https://support.uvdesk.com/en/

You can also visit the forum: https://forums.uvdesk.com/

**Reply by YunusEmreAlps** [reply]
Thank You! 
I found the installation video but not working for me. (https://www.youtube.com/watch?v=O_loE1sqwWA)


![WhatsApp Image 2022-07-29 at 15 05 03](https://user-images.githubusercontent.com/44679555/181754903-ac15cc76-da49-4319-a74c-31503904b2ee.jpeg)
![WhatsApp Image 2022-07-29 at 15 04 32 (1)](https://user-images.githubusercontent.com/44679555/181754920-4a4fa178-0c86-470c-883e-79349bace457.jpeg)

When I click the public file I can't reach the configuration page.

**Reply by PeopleInside** [reply]
Hi @YunusEmreAlps have you found the solution to the not a loading page? 
Seems the error you was getting was a reset connection issue, showed by the browser. Maybe something that was caused by a wrong configuration of .htaccess or server.

**Reply by YunusEmreAlps** [reply]
Thank You @PeopleInside
I used docker and everything was solved on my side.

**Reply by Bamstian** [reply]
Hi @YunusEmreAlps I also use docker but my stack does not work properly. May I ask for your stack? Or could you have a look to my stack and give me a tip what I am missing?

`version: '3'
services:
  db:
    image: mysql:5.7
    volumes:
      - /home/basti/uvdesk/db:/var/lib/mysql
    restart: always
    environment:
      MYSQL_DATABASE: uvdesk
      MYSQL_ROOT_PASSWORD: $MYSQL_ROOT_PW
      MYSQL_USER: uvdesk
      MYSQL_PASSWORD: $MYSQL_PW

  uvdesk:
    image: nuttcorp/uvdesk:latest
    restart: always
    depends_on:
      - db
    tty: true
    environment:
      MYSQL_USER: uvdesk
      MYSQL_PASSWORD: $MYSQL_PW
      MYSQL_ROOT_PASSWORD: $MYSQL_ROOT_PW
      MYSQL_DATABASE: uvdesk
    ports:
      - 6744:80
    volumes:
      - /home/basti/uvdesk:/var/www/html/uvdesk`

**Reply by PeopleInside** [reply]
@Bamstian currently UVdesk free support are suspended since different weeks.
I migrated to a different software so I cannot help.

**Reply by Bamstian** [reply]
Which software do you use now? Kind regards. @PeopleInside

**Reply by PeopleInside** [reply]
I'm using FreeScout. It's not fully free as I need bought some modules but it's not easy to find a good responsive Help Desk system, also free and open source. https://freescout.net/

**Comment by alex23232-tech** [comment]
To run this project, you will need to follow the standard setup process outlined in the repository. For detailed instructions, refer to the README file or the project’s documentation. If you're looking for expert help with recruitment or hiring processes, feel free to check out our [Recruitment Services](https://hrbs.com.pk/recruitment-headhunting/). We specialize in connecting businesses with top talent and can assist you in streamlining your hiring needs. Let me know if you need any more help with the project setup!
